force-app/main/lwcComponentLibrary/classes/singlelookup/LookupResult.cls

```diff
@@ -0,0 +1,74 @@
+public virtual with sharing class LookupResult implements Comparable {
+    @AuraEnabled
+    public String label;
+    @AuraEnabled
+    public String value;
+
+    /*Optional*/
+    @AuraEnabled
+    public String alternativeText;
+    @AuraEnabled
+    public String icon;
+    @AuraEnabled
+    public String imageUrl;
+    @AuraEnabled
+    public String meta;
+
+    public LookupResult(LookupResult result) {
```

Comment:
This is a SAL compatibility issue with one of their code styles. Since this is shared with AC and SAL, this class is not our own and should accommodate their patterns. This is used to create an 'equal but new' instance.